sources/react-app/src/pages/Category.js

```diff
@@ -123,9 +123,16 @@ export default function (props) {
   } = props;
 
   const isTag = match.path === '/tag/:id?';
-  const categoryId = match.params.id;
+  const [categoryId, setCategoryId] = useState();
   let resource = useTaxonomiesResource();
 
+
+  useEffect(() => {
+    if (match.path === '/category/:id?' || match.path === '/tag/:id?') {
+      setCategoryId(match.params.id);
+    }
+  }, [match.params, match.path]);
```

Comment:
Why is the route something other than this at times?

Reply:
Actually it isn't. Removing validation